src/sbml/maddy/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+# This project is licensed under the MIT license. For more information see the
+# LICENSE file.
+
+cmake_minimum_required(VERSION 3.25)
```

Comment:
since maddy requries cmake 3.25 it would be best that all the other requirements are also upgraded currently we still have a wider range that we allow compilation with cmake_minimum_required(VERSION 2.8.4...3.19). We might also have to upgrade the c++ standard to c++14 which is fine with me.